pkgs/by-name/dl/dl-librescore/package.nix

```diff
@@ -9,22 +9,21 @@
 
 buildNpmPackage rec {
   pname = "dl-librescore";
-  version = "0.35.29";
+  version = "0.35.32";
 
   src = fetchFromGitHub {
     owner = "LibreScore";
     repo = "dl-librescore";
     rev = "v${version}";
-    hash = "sha256-DwDlGTFdqAAsEWrhnieuaeYQ0N8COB/7b49xPJackJQ=";
+    hash = "sha256-qFAJlcyvH0lShaZKIAJkXqrY0DRaUvr8V9Ipch1A3kw=";
   };
 
   npmDepsHash = "sha256-5Uc83VdqMwQaTSmzwpBh7x4IKoVPd9MYDXkDvR1fz6Q=";
 
   # see https://github.com/LibreScore/dl-librescore/pull/32
-  # TODO can be removed with next update
```

Comment:
Rather than remove this comment we should open an upstream PR that updates this line:
https://github.com/LibreScore/dl-librescore/blob/ff94de517b49435e393e8b5414270c2ad3a51f89/package-lock.json#L3158

Reply:
done: https://github.com/LibreScore/dl-librescore/pull/144